### PR TITLE
Notification style

### PR DESCRIPTION
**Pull request by dev-matan-tsuberi**
integrate the styling for notifications.

### CODE REVIEW

File: src/components/Notification/Notification.tsx [hunk 1]
```diff
@@ -2,43 +2,68 @@ import * as moment from "moment";
...
+import classNames = require("classnames");
 
 interface IProps {
   operation: IOperation;
 
   close: () => any;
 }
 
-export default class Notification extends React.Component<IProps, null> {
+export default class Notification extends React.Component<IProps, { minimized: boolean }> {
```
Comment (tibetsprague) [inline src/components/Notification/Notification.tsx]: lets create an IState interface for the state

File: src/components/Notification/Notification.tsx [hunk 2]
```diff
@@ -2,43 +2,68 @@ import * as moment from "moment";
...
-          <span>step {step}</span> }
-        <br/>
-        <small className={css.timestamp}>{moment(timestamp).fromNow()}</small>
-        { operation.status !== OperationsStatus.Pending ? <img
-          src="/assets/images/Icon/Close.svg"
-          alt="close"
-          className={css.close}
-          onClick={this.handleClose}/>
-          : ''}
+      <div className={transactionClass} onClick={() => status === OperationsStatus.Pending && minimized ? this.setState({minimized: false}) : null }>
```
Comment (tibetsprague) [inline src/components/Notification/Notification.tsx]: lets pull this out into a separate function

File: src/components/Notification/Notification.tsx [hunk 3]
```diff
@@ -2,43 +2,68 @@ import * as moment from "moment";
...
+        </div>
+        <div className={css.transactionMessage}>
+          <div className={css.clearfix}>
+            <div className={css.left}>
+              <span className={css.pending}>PENDING TRANSACTION</span>
+              <span className={css.success}>TRANSACTION SUCCESSFUL</span>
+              <span className={css.error}>TRANSACTION FAILED</span>
+            </div>
+            <div className={css.right}>
+              <span className={css.pending}>{totalSteps !== 1 ? `Step ${step}` : ''} {totalSteps && totalSteps !== 1 ? `OF ${totalSteps}` : ''}</span>
```
Comment (tibetsprague) [inline src/components/Notification/Notification.tsx]: pull out the steps text into a const

File: src/layouts/AppContainer.tsx [hunk 4]
```diff
@@ -97,35 +97,37 @@ class AppContainer extends React.Component<IProps, null> {
...
-                />),
-              )}
-            </div>
-            <div className={css.background}></div>
-          </div>
-        : <div className={css.loading}>Something weird happened, please contact the DAOstack team...</div>
+        : connectionStatus === ConnectionStatus.Failed
+          ? <NoWeb3Container />
+          : ethAccountAddress === null
+            ? <NoEthAccountContainer />
```
Comment (tibetsprague) [inline src/layouts/AppContainer.tsx]: hmm, i find this indentation more confusing, why do you like it better?

Comment (dev-matan-tsuberi) [reply src/layouts/AppContainer.tsx]: vscode auto format...

File: src/components/Notification/Notification.tsx (hunk 3 above)
Comment (dev-matan-tsuberi) [reply src/components/Notification/Notification.tsx]: updated: https://github.com/daostack/alchemy/pull/52/commits/191f7d3b870fba072c530d61301349f4f3f7ff94

File: src/components/Notification/Notification.tsx (hunk 2 above)
Comment (dev-matan-tsuberi) [reply src/components/Notification/Notification.tsx]: updated: https://github.com/daostack/alchemy/pull/52/commits/191f7d3b870fba072c530d61301349f4f3f7ff94

File: src/components/Notification/Notification.tsx (hunk 1 above)
Comment (dev-matan-tsuberi) [reply src/components/Notification/Notification.tsx]: updated: https://github.com/daostack/alchemy/pull/52/commits/191f7d3b870fba072c530d61301349f4f3f7ff94

**Pull request merged**